src/ethereum-providers/connectors.ts

```diff
@@ -31,21 +44,31 @@ export const CONNECTORS = [
       pollingInterval: 12000,
     },
   },
-  PORTIS_ID
-    ? {
-        id: 'portis',
-        properties: {
-          dAppId: PORTIS_ID,
-          chainId: [100, 4],
-        },
-      }
-    : null,
-].filter(p => p)
+]
+
+if (PORTIS_ID) {
+  CONNECTORS.push({
+    id: 'portis',
+    properties: {
+      dAppId: PORTIS_ID,
+      chainId: [100, 4],
+    },
+  })
```

Comment:
Extract that IF to correct the return type. it was retuning union with null and getting errors on other file use that.